solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LambdaMARTModel.java

```diff
@@ -193,14 +193,12 @@ public LambdaMARTModel(String name, List<Feature> features,
     final List<Object> jsonTrees = getParams().containsKey("trees") ?
         (List<Object>) getParams().get("trees") : null;
 
-    if ((jsonTrees == null) || jsonTrees.isEmpty()) {
-      throw new ModelException("LambdaMARTModel doesn't contain any trees");
-    }
-
-    for (final Object o : jsonTrees) {
-      final Map<String,Object> t = (Map<String,Object>) o;
-      final RegressionTree rt = new RegressionTree(t, fname2index);
-      trees.add(rt);
+    if (jsonTrees != null) {
```

Comment:
Will this null check be needed once you move it into the plugin style set function in the future?